twitchio/user.py

```diff
@@ -693,6 +693,42 @@ async def create_prediction(
         )
         return Prediction(self._http, data[0])
 
+    async def fetch_schedule(
+        self,
+        segment_ids: List[str] = None,
+        start_time: str = None,
```

Comment:
this should really take a Datetime object and internally convert it

Reply:
Changed to datetime and converts to RFC3339 string